include/boost/geometry/algorithms/detail/overlay/get_turn_info_la.hpp

```diff
@@ -261,7 +263,7 @@ struct get_turn_info_linear_areal
                         AssignPolicy::apply(tp, pi, qi, inters);
                         
                         // conditionally handle spikes
-                        if ( ! handle_spikes
+                        if ( ! BOOST_GEOMETRY_CONDITION(handle_spikes)
```

Comment:
Yes, this is why I started to use this convention in all cases, to be consistent.